File: code/js/src/services/Rels.ts
const Rels = {
  SELF: 'self',
  ITEM: 'item',
  HOME: 'home',
  USER_HOME: 'user-home',
  REGISTER: 'register',
  LOGIN: 'login',
  LOGOUT: 'logout',
  USER: 'get-user',
  USERS: 'get-users',
  JOIN_LOBBY: 'join-lobby',
  FIND_MATCH: 'find-match',
  LEAVE_LOBBY: 'leave-lobby',
  GAMES: 'get-games',
  GAME: 'get-game',
  PLAY_GAME: 'play-game',
  LEAVE_GAME: 'leave-game',
  OPPONENT: 'get-opponent',
  BLACK_PLAYER: 'get-black-player',
  WHITE_PLAYER: 'get-white-player',
} as const;

export default Rels;

File: code/jvm/src/main/kotlin/pt/isel/daw/gomoku/http/utils/Rels.kt
package pt.isel.daw.gomoku.http.utils

object Rels {

    const val SELF = "self"
    const val HOME = "home"

    object User {
        const val HOME = "user-home"
        const val REGISTER = "register"
        const val LOGIN = "login"
        const val LOGOUT = "logout"
        const val USER = "get-user"
        const val USERS = "get-users"
    }

    object Lobby {
        const val JOIN = "join-lobby"
        const val FIND_MATCH = "find-match"
        const val LEAVE = "leave-lobby"
    }

    object Game {
        const val GAME = "get-game"
        const val PLAY = "play-game"
        const val GAMES = "get-games"
        const val LEAVE = "leave-game"
        const val OPPONENT = "get-opponent"
        const val WHITE_PLAYER = "get-white-player"
        const val BLACK_PLAYER = "get-black-player"
    }

    object Collection {
        const val COLLECTION = "collection"
        const val ITEM = "item"
        const val NEXT = "next"
        const val PREV = "prev"
    }
}
